bn-apps/cordapp-updates-distribution/design/design.md

```diff
@@ -0,0 +1,135 @@
+![Corda](https://www.corda.net/wp-content/uploads/2016/11/fg005_corda_b.png)
+
+# CordApp distribution service (CDS)
+
+DOCUMENT MANAGEMENT
+---
+
+## Document Control
+
+| Title                | CordApp distribution service                             |
+| -------------------- | ------------------------------------------------------------ |
+| Date                 | 22 Aug 2018                                                |
+| Author               | Ivan Schasny, Mike Hearn |
+| Distribution         | Design Review Board, Product Management, Solutions Engineering, Platform Delivery |
+| Corda target version | OS                                                   |
+
+## HIGH LEVEL DESIGN
+
+### Overview
+
+This proposal describes the architecture of a reference implementation for a CorDapp Distribution Service (CDS), that would allow BNOs to distribute CorDapps updates to Business Network participants. This design targets scenarios when Business Network members maintain their nodes by themselves and can decide on what version of a CorDapp they would like to upgrade to. This design does *not* target scenarios, when upgrade function is centralised and the whole network can be shut down to perform stop-upgrade-restart.
```

Comment:
"CorDapp updates" or "updates to CorDapps"

Reply:
does *not* target scenarios where (no comma and where, not when)

Reply:
Thanks, fixed